.travis.yml

```diff
@@ -0,0 +1,36 @@
+language: node_js
+notifications:
+  email: false
+branches:
+  only:
+  - "/\\d+\\.\\d+\\.\\d+(-\\w+\\.\\d)$/"
+node_js:
+- '8'
+os:
+- linux
+- osx
+git:
+  depth: 5
+matrix:
+  include:
+  - os: linux
+    dist: trusty
+  - os: osx
+install:
+- npm install
+script:
+- npm run build
+before_deploy:
+- git config --global user.email "builds@travis-ci.com"
+- git config --global user.name "Travis CI"
+deploy:
+  provider: releases
+  api_key:
+    secure: P3j+xuXwAMMrLPkPQB7PA9Lqq7YM7nZlqoXiU8KFAcKNeRu+JGKI9Av9LSYKmhaoTsPnlEkeqzXSrfGskCRXuqjd8///niwqij3FguM9xZ1edfmjQbssa4adLQFQvG0v0QCbwdfDCnnahBxD3bwucccoApTpOxzRtXpwyHH36zpCcDfKi5WZJlvPE8Pc5e4wg9KsHoJFmOmzUecHvTSy1cxsCheSgCM5AUlGlD21YGSwkgYSu2a/h/XtUC4WWOrtJY38zQKS6wb8iUWOsj6+2tp50Eis9vb0+Adb1LyCWiwPBzV5k9+f529VsRNYYGxO0jwF+zEYXOh1clGkRyBh2VdV8Cwo+yT0LR0z5PmoNqDTptOXJofIeHYvAUh9Cjz/f3t49sh/kCYtDEGamBCmaGxuaVBr2rNfDhtuKSoPEw+2jb4es5/8+WLRcaUF1O2W+QoIjK9PJDT7jRzo7jJZGbAWM2fepFYLOGRSgfrQ//VOhDFWAIs2Ii0labUHJdkagHa/gq5qcPyO/fXzomrNb4ljY10HOEOPkZLf4LwD/0rtyJXq5QPZuY2r8bDf6v3+3k6/fDDt6qyNAlUYRhnDOvuKIEirkb9VQdGc9uSed/lLAXqdwb19oce0woM6nVj/VErWNSI4PMkK9NOEysMlevPJd+IgBZ9PVyvMmxlcRV0=
+  file_glob: true
+  file: out/make/*
```

Comment:
I think we should rename the output files so that they match how weve been doing it.

Perhaps this is just a config in the electron build?

but MACOS_.....zip, LINUX_DEB.....deb